test/unit/ngsiv2/HTTP_reveice_measures-test2.js

```diff
@@ -385,25 +350,7 @@ describe('HTTP: Measure reception ', function () {
                 .matchHeader('fiware-service', 'smartgondor')
                 .matchHeader('fiware-servicepath', '/gardens')
                 .post(
-                    '/v2/entities?options=upsert',
-                    utils.readExampleFile('./test/unit/ngsiv2/contextRequests/timeInstantMeasures1b.json')
-                )
-                .reply(204);
-
-            contextBrokerMock
-                .matchHeader('fiware-service', 'smartgondor')
-                .matchHeader('fiware-servicepath', '/gardens')
-                .post(
-                    '/v2/entities?options=upsert',
-                    utils.readExampleFile('./test/unit/ngsiv2/contextRequests/timeInstantMeasures2b.json')
-                )
-                .reply(204);
-
-            contextBrokerMock
-                .matchHeader('fiware-service', 'smartgondor')
-                .matchHeader('fiware-servicepath', '/gardens')
-                .post(
-                    '/v2/entities?options=upsert'
+                    '/v2/op/update'
                     // FIXME: Mock about h atttribute + current timestamp
```

Comment:
Are this FIXME expected to be solved in this PR?

It seem that in other places in this PR (L426) a similar FIXME has been solved.

Reply:
No fix about this is expected in this PR.

Reply:
Ah, I din't realize this is white (not green, not red), so it's existing code.

NTC